src/System.Reflection.DispatchProxy/src/System.Reflection.DispatchProxy.csproj

```
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkMinimum);netstandard2.0</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <StrongNameKeyId>Open</StrongNameKeyId>
+    <IsPackable>false</IsPackable>
+    <VersionPrefix>4.7.1</VersionPrefix>
```

```suggestion
    <StrongNameKeyId>Microsoft</StrongNameKeyId>
    <VersionPrefix>4.7.1</VersionPrefix>
```

This should fix the compat error.  See https://github.com/dotnet/arcade/blob/2fb543a45580400a559b5ae41c96a815ea14dac5/src/Microsoft.DotNet.Arcade.Sdk/tools/StrongName.targets#L31C25-L31C40 for the decoder ring.

When copying further assemblies over you can migrate this from here:
https://github.com/dotnet/corefx/blob/b58ef0a19eca6269799664d6288be14b7f0cef3e/src/System.Reflection.DispatchProxy/Directory.Build.props#L6

---

Fixed. I actually modified the property in line 6, it was set to Open.